src/com/rcv/RawElectionConfig.java

```diff
@@ -60,6 +60,8 @@
     public String filePath;
     // column where rankings data begins
     public Integer firstVoteColumnIndex;
+    // column containing precinct (if any)
+    public Integer precinctColumnIndex;
```

Comment:
I'm unclear on whether or not precintColumnIndex is optional or not; it seems like it's required, but your comment says "if any" here.

Reply:
I made it optional for the time being.